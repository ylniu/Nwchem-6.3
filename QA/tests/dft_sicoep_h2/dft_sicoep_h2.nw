title "h2_sicoep"

start h2_sicoep

echo


geometry units angstrom
symmetry group d2h
  H     0.0  0.0  0.37
end


basis "ao basis" spherical print
 H library "DZ (Dunning)"
end


dft
 xc slater 1.0 vwn_5 1.0
 sic oep
 mult 1
 print "SIC information"
end

task dft energy

