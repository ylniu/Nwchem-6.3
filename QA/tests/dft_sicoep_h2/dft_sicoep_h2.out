HP-MPI licensed for ISV application.
All connections between all procs tested: SUCCESS
 argument  1 = dft_sicoep_h2.nw



============================== echo of input deck ==============================
title "h2_sicoep"

start h2_sicoep

echo


geometry units angstrom
symmetry group d2h
  H     0.0  0.0  0.37
end


basis "ao basis" spherical print
 H library "DZ (Dunning)"
end


dft
 xc slater 1.0 vwn_5 1.0
 sic oep
 mult 1
 print "SIC information"
end

task dft energy

================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = cu0login1
    program       = /hptc_cluster/scicons/apps/nwchem-6.0.oct19/bin/LINUX64/nwchem
    date          = Mon Nov  1 15:15:30 2010

    compiled      = Thu_Oct_28_07:10:53_2010
    source        = /home/scicons/user/kurt/nwchem-6.0-release-pgf90-final/
    nwchem branch = 6.0
    input         = dft_sicoep_h2.nw
    prefix        = h2_sicoep.
    data base     = ./h2_sicoep.db
    status        = startup
    nproc         =        1
    time left     =     -1s



           Memory information
           ------------------

    heap     =  104857601 doubles =    800.0 Mbytes
    stack    =  104857601 doubles =    800.0 Mbytes
    global   =  209715200 doubles =   1600.0 Mbytes (distinct from heap & stack)
    total    =  419430402 doubles =   3200.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                                     h2_sicoep
                                     ---------

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 H                    1.0000     0.00000000     0.00000000     0.37000000
    2 H                    1.0000     0.00000000     0.00000000    -0.37000000

      Atomic Mass 
      ----------- 

      H                  1.007825


 Effective nuclear repulsion energy (a.u.)       0.7151043905

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             D2h       
 Group number             26
 Group order               8
 No. of unique centers     1

      Symmetry unique atoms

     1


            XYZ format geometry
            -------------------
     2
 geometry
 H                     0.00000000     0.00000000     0.37000000
 H                     0.00000000     0.00000000    -0.37000000

                      Basis "ao basis" -> "" (spherical)
                      -----
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.92406000E+01  0.032828
  1 S  2.89920000E+00  0.231208
  1 S  6.53400000E-01  0.817238

  2 S  1.77600000E-01  1.000000



 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 H                        DZ (Dunning)               2        2   2s



                                 NWChem DFT Module
                                 -----------------


                                     h2_sicoep


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     2
          No. of electrons :     2
           Alpha electrons :     1
            Beta electrons :     1
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: on ; symmetry adaption is: on 
          Maximum number of iterations:  30
          AO basis - number of functions:     4
                     number of shells:     4
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
                        Slater Exchange Functional  1.000 local    
                      VWN V Correlation Functional  1.000 local    

              SIC/OEP without localized orbitals

             Grid Information
             ----------------
          Grid used for XC integration:  medium    
          Radial quadrature: Mura-Knowles        
          Angular quadrature: Lebedev. 
          Tag              B.-S. Rad. Rad. Pts. Rad. Cut. Ang. Pts.
          ---              ---------- --------- --------- ---------
          H                   0.35       45           5.0       434
          Grid pruning is: on 
          Number of quadrature shells:    45
          Spatial weights used:  Erf1

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -0.99527373

      Non-variational initial energy
      ------------------------------

 Total energy =      -0.845122
 1-e energy   =      -2.232107
 2-e energy   =       0.671881
 HOMO         =      -0.574585
 LUMO         =       0.194761


      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------

  Numbering of irreducible representations: 

     1 ag          2 au          3 b1g         4 b1u         5 b2g     
     6 b2u         7 b3g         8 b3u     

  Orbital symmetries:

     1 ag          2 b1u         3 ag          4 b1u     

   Time after variat. SCF:      0.0
   Time prior to 1st pass:      0.0

 #quartets = 3.700D+01 #integrals = 3.700D+01 #direct =  0.0% #cached =100.0%


 Integral file          = ./h2_sicoep.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   = ******
 No. of bits per label  =      8        No. of bits per value  =     64


 Grid_pts file          = ./h2_sicoep.gridpts.0
 Record size in doubles =  12289        No. of grid_pts per rec  =   3070
 Max. records in memory =      8        Max. recs in file   =  44841235


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):      104.63           104627568
          Stack Space remaining (MW):      104.86           104857452

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1     -1.1310554038 -1.85D+00  1.23D-02  2.92D-03     0.1
 d= 0,ls=0.0,diis     2     -1.1316150873 -5.60D-04  2.35D-03  1.60D-04     0.1
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 d= 0,ls=0.0,diis     3     -1.1316476204 -3.25D-05  1.37D-09  5.43D-17     0.1
  Starting SIC by orbital..
  Orbital                         1
  SIC approximation in Coulomb term
  tot_sic_coul, tot_sic_xc, tot_sic:   -0.6526596329736013      
   0.6085936654696920       -4.4065967503909342E-002
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 d= 0,ls=0.0,diis     4     -1.1757135879 -4.41D-02  1.37D-09  9.95D-04     0.7
  Starting SIC by orbital..
  Orbital                         1
  SIC approximation in Coulomb term
  tot_sic_coul, tot_sic_xc, tot_sic:   -0.6526596315765993      
   0.6085936641672020       -4.4065967409397389E-002
  Singularity in Pulay matrix. Error and Fock matrices removed. 
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 d= 0,ls=0.0,diis     5     -1.1757135878  9.45D-11  7.11D-03  9.95D-04     1.0
  Starting SIC by orbital..
  Orbital                         1
  SIC approximation in Coulomb term
  tot_sic_coul, tot_sic_xc, tot_sic:   -0.6599372736506710      
   0.6153804027708804       -4.4556870879790655E-002
 d= 0,ls=0.0,diis     6     -1.1759074308 -1.94D-04  1.23D-03  4.36D-05     1.2
  Starting SIC by orbital..
  Orbital                         1
  SIC approximation in Coulomb term
  tot_sic_coul, tot_sic_xc, tot_sic:   -0.6586732118027181      
   0.6142013987601629       -4.4471813042555208E-002
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 d= 0,ls=0.0,diis     7     -1.1759163219 -8.89D-06  2.55D-07  1.86D-12     1.4
  Starting SIC by orbital..
  Orbital                         1
  SIC approximation in Coulomb term
  tot_sic_coul, tot_sic_xc, tot_sic:   -0.6586729505352246      
   0.6142011550821737       -4.4471795453050911E-002
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 d= 0,ls=0.0,diis     8     -1.1759163219 -2.58D-13  1.08D-11  3.35D-21     1.6


         Total DFT energy =       -1.175916321855
      One electron energy =       -2.500325197940
           Coulomb energy =        1.317345901070
    Exchange-Corr. energy =       -0.708041415525
 Nuclear repulsion energy =        0.715104390540

 Numeric. integr. density =        1.999999947139

     Total iterative time =      1.6s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=2.000000D+00  E=-6.197841D-01  Symmetry=ag
              MO Center=  0.0D+00,  0.0D+00, -1.1D-16, r^2= 7.0D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1     -0.317741  1 H  s                  3     -0.317741  2 H  s          
     4     -0.277668  2 H  s                  2     -0.277668  1 H  s          

 Vector    2  Occ=0.000000D+00  E=-1.293659D-01  Symmetry=b1u
              MO Center=  0.0D+00,  0.0D+00,  2.2D-16, r^2= 1.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.543399  1 H  s                  4     -1.543399  2 H  s          
     1      0.193961  1 H  s                  3     -0.193961  2 H  s          

 Vector    3  Occ=0.000000D+00  E= 3.939615D-01  Symmetry=ag
              MO Center=  0.0D+00,  0.0D+00, -5.3D-16, r^2= 1.5D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.804804  2 H  s                  1      0.804804  1 H  s          
     4     -0.716010  2 H  s                  2     -0.716010  1 H  s          

 Vector    4  Occ=0.000000D+00  E= 9.275696D-01  Symmetry=b1u
              MO Center=  0.0D+00,  0.0D+00, -2.8D-17, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -1.483313  1 H  s                  4      1.483313  2 H  s          
     3     -1.135934  2 H  s                  1      1.135934  1 H  s          


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
           0.985408360325           0.000000000000           0.000000000000
           0.000000000000           0.985408360325           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0      0.000000     -1.000000     -1.000000      2.000000

     1   1 0 0      0.000000      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000      0.000000
     1   0 0 1      0.000000      0.000000      0.000000      0.000000

     2   2 0 0     -1.447063     -0.723531     -0.723531      0.000000
     2   1 1 0      0.000000      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000      0.000000
     2   0 2 0     -1.447063     -0.723531     -0.723531      0.000000
     2   0 1 1      0.000000      0.000000      0.000000      0.000000
     2   0 0 2     -1.128147     -1.052952     -1.052952      0.977757


 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        1.6s     wall:        2.0s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  225      225     4391      540     2127        0        0        0     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             1.61e+05 3.54e+04 8.39e+04 0.00e+00 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 4992 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        24	        53
	current total bytes		         0	         0
	maximum total bytes		   1840264	  22509744
	maximum total K-bytes		      1841	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        1.6s     wall:        2.2s
